.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Description, Motivation and Context
+
+Please include a summary of the change or what problem does it solve. Please also include relevant motivation and context.
+
+<!--- If it relates to an open issue, please link to the issue here.
+e.g.
+[issue 414](https://github.com/replicatedhq/troubleshoot.sh/issues/414)
```

Comment:
Could suggest Github format links - e.g. for the example it would just be #414, e.g. "Fixes: #414"